docs/index.md

```diff
@@ -9,7 +9,7 @@ main: () = {
 
 ## What is Cpp2?
 
-"Cpp2," short for "C++ syntax 2," is my ([Herb Sutter's](https://github.com/hsutter)) personal project to try to make writing ordinary C++ types/functions/objects be much **simpler and safer**, without breaking backward compatibility. Bjarne Stroustrup said it best:
+"Cpp2", short for "C++ syntax 2", is my ([Herb Sutter's](https://github.com/hsutter)) personal project to try to make writing ordinary C++ types/functions/objects be much **simpler and safer**, without breaking backward compatibility. Bjarne Stroustrup said it best:
```

Comment:
Isn't it U.S. convention to put the comma inside the quotes?

Reply:
For this one I think I'll go back, but you're right there are exceptions. Please point them out as you spot debatable ones -- thanks!